src/components/pages/stats/StatsPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import UniqueVisitsCharts from 'components/molecules/charts/uniqueVisitChart/UniqueVisitChart';
+import {
+  Flex,
+  Heading,
+  Card,
+  CardBody,
+  CardHeader,
+  useMediaQuery,
+} from '@chakra-ui/react';
+import VisitLastMonthChart from 'components/molecules/charts/pageVisitsLastMonthChart/VistsLastMonthChart';
+
+const UniqueChartBox: React.FC = () => {
+  return (
+    <Card
+      w="95%"
+      alignSelf="center"
+      variant={'outline'}
+      colorScheme="blue"
+      backgroundColor="unset">
+      <CardHeader margin={0}>Unike brukere</CardHeader>
+      <CardBody>
+        <UniqueVisitsCharts />
+      </CardBody>
+    </Card>
+  );
+};
+
+const PageVisitChartBox: React.FC = () => {
+  return (
+    <Card
+      w="90%"
+      mt={25}
+      alignSelf="center"
+      variant={'outline'}
+      colorScheme="blue"
+      backgroundColor="unset">
+      <CardHeader margin={0}>Mest besøkte sider siste 30 dager</CardHeader>
+      <CardBody>
+        <VisitLastMonthChart />
+      </CardBody>
+    </Card>
+  );
+};
+const StatsPage: React.FC = () => {
+  // since media query is false then true we have to ensure that
+  // the smallest value is sat first as the react chart does not resize when parent gets smaller
+  const [width, setWidth] = useState('50%');
+  const [isLargerThan800] = useMediaQuery('(min-width: 768px)');
+
+  useEffect(() => {
+    setWidth(isLargerThan800 ? '50%' : '100%');
+  }, [isLargerThan800]);
+
+  return (
+    <Flex w="100vw" h="100%" flexDir="column" textAlign="center" mb={50}>
+      <Heading mt={2.5}>Statistikk</Heading>
+      <UniqueChartBox />
+      <Flex w={width} h="50%" justifyContent={'center'}>
```

Comment:
Using chakra breakpoints here could be a better way than using media queries and useEffect in this case.